contributing-guides/style-guide.zh.md

```diff
@@ -73,6 +73,32 @@ tldr -f {{page.md}}
 
 在命令描述中，如果出现了技术性的专有名词，请用 `反引号` 括起来：
 
-1. 路径，例如 `package.json`，`/etc/package.json`。
-2. 扩展名，例如 `.dll`。
-3. 命令，例如 `ls`。
+1. 路径，例如 `package.json`，`/etc/package.json`.
+2. 扩展名，例如 `.dll`.
+3. 命令，例如 `ls`.
+
+## 中西文混排规则
+
+中文、西文、阿拉伯数字写在同一个句子时，需要注意排版。
+
+以下规则适用于中文（zh）和繁体中文（zh_TW）：
+
+1. 在西文单词和数字前 / 后放置一个空格。
+   例如：`列出所有 docker 容器`而不是`列出所有docker容器`。 
+   例如：`宽度为 50 个字`而不是`宽度为50个字`。
+2. 除了度数和百分比，在数字和单位之间留一个空格。
+   例如：`容量 50 MB`而不是`容量 50MB`。
+   对于度数和百分比：使用`50°C`和`50%`而不是`50 °C`和`50 %`
+3. 不要在全角标点符号前 / 后放置空格。
```

Comment:
The same. Chinese user can understand it without '/'. maybe better to remove it.